src/utils/waitForPostgres.ts

```diff
@@ -1,7 +1,8 @@
 import type { IIoK8sApiCoreV1Container } from "kubernetes-models/_definitions/IoK8sApiCoreV1Container";
 
-// renovate: datasource=docker depName=registry.gitlab.factory.social.gouv.fr/socialgouv/docker/wait-for-postgres versioning=5.0.1
-const SOCIALGOUV_DOCKER_WAIT_FOR_PG_VERSION = "5.0.1";
+// renovate: datasource=docker depName=ghcr.io/socialgouv/docker/wait-for-postgres versioning=6.0.1
+const SOCIALGOUV_DOCKER_IMAGE = "ghcr.io/socialgouv/docker/wait-for-postgres";
+const SOCIALGOUV_DOCKER_VERSION = "6.0.1";
```

Comment:
The renovate commit should be next to the version variable 
```suggestion
const SOCIALGOUV_DOCKER_IMAGE = "ghcr.io/socialgouv/docker/wait-for-postgres";
// renovate: datasource=docker depName=ghcr.io/socialgouv/docker/wait-for-postgres versioning=6.0.1
const SOCIALGOUV_DOCKER_VERSION = "6.0.1";
```